src/net.cpp

```diff
@@ -1513,18 +1513,12 @@ void CConnman::SocketHandler()
         }
     }
 
+    const NodesSnapshot snap{*this, /*shuffle=*/false};
```

Comment:
> What do you think about always shuffling?

I do not have a strong opinion. My main point with the current code is to preserve behavior - it was not shuffled before, it will not be shuffled with this PR. This PR has a different purpose.

In general, I try to avoid such unnecessary behavioral changes. _They may have very subtle effects._

Maybe it is ok to always shuffle and remove the boolean argument from the `NodesSnapshot` constructor? What do other reviewers think?

Reply:
Slight preference to keep behavior here and simplify in a follow-up.